e2e/verify/verify.go

```diff
@@ -102,7 +102,7 @@ func (c ctx) singularityVerifyKeyNum(t *testing.T) {
 			e2e.AsSubtest(tt.name),
 			e2e.WithProfile(e2e.UserProfile),
 			e2e.WithCommand("verify"),
-			e2e.WithArgs("--json", tt.imagePath),
+			e2e.WithArgs("--all", "--json", tt.imagePath),
```

Comment:
Should we change this test from `singularityVerifyKeyNum()` to `singularityVerifyAll()`?

Reply:
I changed it to `singularityVerifyAllKeyNum()`, and left the `...KeyNum()` since that is what we are actually testing, the number of keys.